main/connect_api.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.shortcuts import redirect
+from urllib.parse import urlencode
+import requests
+
+
+API_KEY = "9f97e9f457aa4379ba2cb4c32072aec4"
```

Comment:
A reminder that we should keep API in env var after we all test out the functionality

Reply:
Right, I fixed this. Thanks for the reminder.